footylib/footylib.py

```diff
@@ -166,64 +196,61 @@ def _populate(self, info, division):
         except KeyError:
             self.logger.exception("Got an exception on Matches.")
 
-    def __string_to_datetime(self, date, time):
-        dutch_datetime = '{} {}'.format(date, time).split()
-        english_datetime = self.__dutch_to_english_reference(dutch_datetime[0])
-        dutch_datetime[0] = english_datetime
-        english_datetime = " ".join(dutch_datetime)
+    @property
+    def visiting_team(self):
```

Comment:
Maybe it would be wise to save this in an internal variable instead of recreating it every time. The same goes for home_team.